src/overlays/actors/ovl_Bg_Numa_Hana/z_bg_numa_hana.h

```diff
@@ -3,15 +3,41 @@
 
 #include "global.h"
 
+#define BG_NUMA_HANA_GET_TYPE(thisx) ((thisx)->params & 1)
+#define BG_NUMA_HANA_SWITCH_FLAG(thisx) (((thisx)->params >> 8) & 0x7F)
+
+typedef enum {
+    /* 0 */ BG_NUMA_HANA_TYPE_NORMAL,
+    /* 1 */ BG_NUMA_HANA_TYPE_OPEN_FLOWER_COLLISION,
+} BgNumaHanaType;
+
 struct BgNumaHana;
 
 typedef void (*BgNumaHanaActionFunc)(struct BgNumaHana*, GlobalContext*);
 
+typedef struct {
+    /* 0x00 */ Vec3f pos;
+    /* 0x0C */ Vec3s rot;
+    /* 0x12 */ char unk_12[0x2];
```

Comment:
```suggestion
    /* 0x12 */ UNK_TYPE1 unk_12[0x2];
```

Reply:
I was almost certain I did this, but apparently not. Thanks for catching it.